src/main/java/com/futurcollars/lesson11/zad3/LengthOfTheText.java

```diff
@@ -0,0 +1,14 @@
+package com.futurcollars.lesson11.zad3;
+
+public class LengthOfTheText {
```

Comment:
Nie przyjmuj danych w konstruktorze, niech będzie to prosta klasa w stylu TextLenghtService z jedną metodą publiczną przyjmującą tekst i zwracającą jego długość